templates/checkouts/new.php

```diff
@@ -37,7 +37,7 @@
     <script src="https://js.braintreegateway.com/web/dropin/1.9.2/js/dropin.min.js"></script>
     <script>
       var form = document.querySelector('#payment-form');
-      var client_token = "<?php echo(Braintree\ClientToken::generate()); ?>";
+      var client_token = "<?php echo($client_token); ?>";
```

Comment:
I don't know PHP too well so I am probably missing something, but is `$client_token` defined or passed in here from somewhere?

Reply:
It's done via the slim framework. It's actually defined here: https://github.com/braintree/braintree_slim_example/pull/11/files#diff-828e0013b8f3bc1bb22b4f57172b019dR23

For some reason, we weren't using the variable that we had created when the repo was first made. (probably because we couldn't figure out how to interpolate it, I had to dig a long time to figure out that slim just automagically creates it)

Reply:
Cool, looks 👍 to me